domain/src/main/java/org/oppia/android/domain/topic/ModifyLessonProgressController.kt

```diff
@@ -0,0 +1,109 @@
+package org.oppia.android.domain.topic
```

Comment:
This file has to go in a separate devoptions package in domain module.

Reply:
![Screenshot from 2021-07-05 22-51-32](https://user-images.githubusercontent.com/63469151/124503644-9b6f2800-dde3-11eb-828e-fead4e12da59.png)
Actually, @BenHenning mentioned to use this `devoptions` package specifically for the UI part only. That's why I put this controller in the `topic` package. Should I move it to `devoptions` package in `domain` module? @Sarthak2601

Reply:
Moved to `devoptions` package. PTAL.